besu/src/test/java/org/hyperledger/besu/controller/TargetingGasLimitCalculatorTest.java

```diff
@@ -48,11 +52,13 @@ public void verifyGasLimitReachesTarget() {
 
   @Test
   public void verifyNoNegative() {
+    // gas limit less or equal than GASLIMIT_BOUND_DIVISOR (1024) can't not be change because of
+    // delta rule
     final long target = 0L;
-    final long offset = TargetingGasLimitCalculator.ADJUSTMENT_FACTOR / 2;
+    final long current = TargetingGasLimitCalculator.GASLIMIT_BOUND_DIVISOR;
     TargetingGasLimitCalculator targetingGasLimitCalculator =
         new TargetingGasLimitCalculator(target);
-    assertThat(targetingGasLimitCalculator.nextGasLimit(target + offset)).isEqualTo(target);
+    assertThat(targetingGasLimitCalculator.nextGasLimit(current)).isEqualTo(current);
```

Comment:
`GasLimitRangeAndDeltaValidationRule` checks that the min gas limit is above the divisor and doesn't accept it. Your ammendent of this test seems to suggest that it should be allowed, but it causes the adjustment to be 0 in perpetuity. Do you have a source for this behavior? I see that when I search `gas limit` here, it seems to say that `5000` is the minimum: 
chrome-extension://gfbliohnnapiefjpjlpjnehglfpaknnc/pages/pdf_viewer.html?r=https://ethereum.github.io/yellowpaper/paper.pdf

Reply:
Yes this would never happen in the integrated scenario, but this test case is originally try to test the negative situation (which won't happen). Do you suggest to remove this test case?

Reply:
I think we should replace it with a test that tests the 5000 gas scenario

Reply:
(ensures that it doesn't go below that value)

Reply:
`5000` is based on the consensus algorithm (ex [IBFT](https://github.com/hyperledger/besu/blob/master/consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/IbftBlockHeaderValidationRulesetFactory.java))
But I can change `TargetingGasLimitCalculator` to enforce a `5000` minimum

Reply:
Oh yeah, I should clarify that I meant a general "minimum" like in `GasLimitRangeAndDeltaValidationRule`, not necessarily 5000 hardcoded. As you can see, the `TargetingGasLimitCalculator` should rely on the validation rule class heavily since all that logic is already there.